src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -170,6 +171,86 @@
             :field-value field-value}]
           [customizer-select-button {:open open :selected-option selected-option}]]]))))
 
+(defn- customizer-multi-select-modal
```

Comment:
Imo changes like this are greatly appreciated and a benefit for the preview screens. With that said, imo it's best that we collaborate with 1 or 2 developers when adding features to the preview screens as these are changes that will affect us all and more input might lead to a better outcome. Perhaps this could benefit from being shared in the chat explaining the idea and gather some feedback before being implemented.
Apologies if this was already done. 👍 

One thing I would say is that this preview file is getting rather large and perhaps it's better we put it into a folder and make these as separate utils we then export from preview/core.cljs. This could be done beyond this pr of course 👍

Reply:
That's a good point, should've dropped a message in #general before starting on it 👍 . Should I still drop a message there for feedback? Also i'll make an issue out of splitting the preview/core.

Reply:
up to you. I think no harm in doing it. Or at least if you could provide a description of what this is, how it works etc then it will be easier to understand the implementation

Reply:
yep, will leave a description as a comment on the PR.

Reply:
@clauxx I don't see see anything in the description. anyway at this point maybe it's a good idea to drop a message with this info into discord as it will probably go unnoticed and perhaps other devs will need to make use of this and otherwise wouldn't know it exists 👍